packages/backend/src/tokens/jwt.test.ts

```diff
@@ -91,8 +91,8 @@ export default (QUnit: QUnit) => {
       assert.propEqual(payload, mockJwtPayload);
     });
 
-    todo('returns the reason of the failure when verifications fail', assert => {
-      assert.true(true);
-    });
+    // todo('returns the reason of the failure when verifications fail', assert => {
+    //   assert.true(true);
+    // });
```

Comment:
🙃 i have merged a PR to comment out the todos. Could you please rebase to take the changes?